doc/user/quick-sql-grammar.md

```diff
@@ -485,6 +483,7 @@ columnDirective::= '/'
       |'between'
       |'hidden'|'invisible'
```

Comment:
You removed `hidden|invisible` above, but it still exists here. Why was it removed in the first place again?

Reply:
fixed